morpheus/cli.py

```diff
@@ -306,7 +310,11 @@ def run(ctx: click.Context, **kwargs):
     pass
 
 
-@click.group(chain=True, short_help="Run the inference pipeline with a NLP model", cls=AliasedGroup, **command_kwargs)
+@click.group(chain=True,
+             short_help="Run the inference pipeline with a NLP model",
+             no_args_is_help=True,
```

Comment:
does "args" include the stages, such as `from-file`, or just the `--` flags? If it's just the flags, then this might be problematic, since in the issue we use
```
morpheus run pipeline-nlp \
             from-file --filename=data/pcap_dump.jsonlines \
             deserialize
```
where pipeline-nlp has no flags. But I assume `from-file` counts as an "arg", and this is fine.

Reply:
This only applies to the last command in the chain. So this would not trigger the help for `pipeline-nlp`:
```
morpheus run pipeline-nlp \
             from-file --filename=data/pcap_dump.jsonlines \
             deserialize
```
But this would:
```
morpheus run pipeline-nlp
```
Since `morpheus run pipeline-nlp` would be an invalid pipeline with no commands, it would be better to just print the help info. The `no_args_is_help` option should be enabled for all `@cli.group` decorators, especially ones with `@cli.group(chain=True)`